### DISCUSSION TITLE
希望增加简介、评论区链接点击跳转功能

### DISCUSSION THREAD

**Discussion by yun-Tong** [the1812/Bilibili-Evolved#1928] [综合讨论 / General]
希望可以在简介或评论区或其他地方增加直接点击链接跳转的功能
默认情况下简介放外链点击没有任何反应
希望增加点击就直接跳转外链的功能

**Comment by FoundTheWOUT** [comment]
B站不做这个是有理由的，它就是想让你自己去复制粘贴。国内许多网站也没有直跳服务，一般都加一层确认跳转。这都是为了给自己规避一些不必要的风险。所以我认为该插件能做的很有限，因为谁也不想承担这一部分风险。

**Reply by Geeyun-JY3** [reply]
提供这种功能的脚本和插件都不少，没啥风险的。
#707 也都打上 tag 了。

**Comment by Geeyun-JY3** [comment]
部分重复了：#707

通用型的文字转链接类脚本和插件都不少，如 [Text To link](https://greasyfork.org/scripts/342)（目前有兼容 bug：#1889）、[Linkify Plus Plus](https://greasyfork.org/scripts/4255)（也有兼容 bug：https://github.com/the1812/Bilibili-Evolved/issues/1242#issuecomment-732125449 ）。

**Comment by Geeyun-JY3** [comment]
#5197 实现了简介的：https://github.com/the1812/Bilibili-Evolved/releases/tag/v2.10.1-preview
